server/valgrind-out.txt

```diff
@@ -1,131 +1,160 @@
+--3734025-- REDIR: 0x4913950 (libc.so.6:stpcpy) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--3734025-- REDIR: 0x49133f0 (libc.so.6:memchr) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--3734025-- REDIR: 0x4912b00 (libc.so.6:strspn) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--3734025-- REDIR: 0x49137d0 (libc.so.6:mempcpy) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--3734025-- REDIR: 0x4913ac0 (libc.so.6:strncasecmp) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--3734025-- REDIR: 0x4914f80 (libc.so.6:rawmemchr) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--3734025-- REDIR: 0x4a07610 (libc.so.6:__strrchr_avx2) redirected to 0x484e810 (rindex)
+--3734025-- REDIR: 0x4a0af80 (libc.so.6:__memset_avx2_unaligned_erms) redirected to 0x4852770 (memset)
+--3734025-- REDIR: 0x490f0a0 (libc.so.6:malloc) redirected to 0x4848820 (malloc)
+--3734025-- REDIR: 0x490f3e0 (libc.so.6:free) redirected to 0x484b210 (free)
+--3734025-- REDIR: 0x4912fc0 (libc.so.6:__GI_strstr) redirected to 0x4853ae0 (__strstr_sse2)
+--3734025-- REDIR: 0x4a032c0 (libc.so.6:__memchr_avx2) redirected to 0x484fd50 (memchr)
+--3734025-- REDIR: 0x4a0a7c0 (libc.so.6:__memcpy_avx_unaligned_erms) redirected to 0x4852880 (memmove)
+--3734025-- REDIR: 0x4a07180 (libc.so.6:__strchr_avx2) redirected to 0x484e9f0 (index)
+--3734025-- REDIR: 0x490f740 (libc.so.6:realloc) redirected to 0x484dc50 (realloc)
+--3734025-- REDIR: 0x4a077e0 (libc.so.6:__strlen_avx2) redirected to 0x484ed10 (strlen)
+--3734025-- REDIR: 0x4a03a80 (libc.so.6:__memcmp_avx2_movbe) redirected to 0x4852010 (bcmp)
+--3734025-- REDIR: 0x4910520 (libc.so.6:calloc) redirected to 0x484d9d0 (calloc)
+--3734025-- REDIR: 0x4a07400 (libc.so.6:__strchrnul_avx2) redirected to 0x4853330 (strchrnul)
+--3734025-- REDIR: 0x4a02940 (libc.so.6:__strcmp_avx2) redirected to 0x484fbd0 (strcmp)
+--3734025-- REDIR: 0x4a0a780 (libc.so.6:__mempcpy_avx_unaligned_erms) redirected to 0x4853440 (mempcpy)
+--3734025-- REDIR: 0x4a08cb0 (libc.so.6:__strcpy_avx2) redirected to 0x484ee60 (strcpy)
+--3734025-- Reading syms from /usr/lib/x86_64-linux-gnu/libgcc_s.so.1
+--3734025--    object doesn't have a symbol table
+--3734025-- REDIR: 0x4a099a0 (libc.so.6:__stpcpy_avx2) redirected to 0x4852130 (stpcpy)
+--3734025-- Discarding syms at 0x5e9b660-0x5eb1805 in /usr/lib/x86_64-linux-gnu/libgcc_s.so.1 (have_dinfo 1)
+==3734025== 
+==3734025== HEAP SUMMARY:
+==3734025==     in use at exit: 2,720 bytes in 19 blocks
+==3734025==   total heap usage: 21,488 allocs, 21,469 frees, 62,379,247 bytes allocated
+==3734025== 
+==3734025== Searching for pointers to 19 not-freed blocks
+==3734025== Checked 8,493,656 bytes
+==3734025== 
+==3734025== 136 bytes in 1 blocks are still reachable in loss record 1 of 3
+==3734025==    at 0x4848899: malloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
+==3734025==    by 0x10B64B: main (in /home/student/AESD/assignment-2-Induja21/server/aesdsocket)
```

Comment:
Probably a real issue - use `-g` compile option to find the specific line number where this is occurring.